wpeverest/everest-forms
# Fix -  Entries pictorial design issue and top spacing of search field during loading time and left, right spacing of graph section

### All Submissions:






### Changes proposed in this Pull Request:



This PR fixes Entries pictorial design issue fixing in Enties page .
![image](https://user-images.githubusercontent.com/66351929/108209473-7474ca00-7152-11eb-8434-3660d0f75805.png)
And top spacing of during loading stage
![image](https://user-images.githubusercontent.com/66351929/108355026-5cfd1600-7212-11eb-9477-832a01e9a6ff.png)
And left right spacing of pictorial design.
![image](https://user-images.githubusercontent.com/66351929/108716819-afa53d80-7544-11eb-88a4-f7f0ec42c396.png)



### How to test the changes in this Pull Request:

1. Checkout to this branch
2. Navigate to Entries tab
3.Please check if the design issue still exist or not

### Types of changes:




### Other information:




### Changelog entry
> Fix -  Entries pictorial design issue and top spacing of search feild during loading time and left right spacing of graph section.
